src/sdltiles.cpp

```diff
@@ -1419,8 +1478,13 @@ int HandleDPad()
  */
 long sdl_keysym_to_curses( SDL_Keysym keysym )
 {
-
-    if( get_option<bool>( "DIAG_MOVE_WITH_MODIFIERS" ) ) {
+    std::string arrow_keys_vs_modifiers = get_option<std::string>( "ARROW_KEYS_VS_MODIFIERS" );
+    if ( arrow_keys_vs_modifiers == "numpad" ) {
+        if( keysym.mod & KMOD_CTRL && sdl_keycode_is_arrow( keysym.sym ) )
+            return handle_arrow_combo( keysym.sym );
+        else
```

Comment:
Official code style (see doc/CODE_STYLE.md) is to always have brackets even around one conditional statements.